src/compiler/declarationEmitter.ts

```diff
@@ -913,6 +916,10 @@ namespace ts {
 
             emitJsDocComments(node);
             emitModuleElementDeclarationFlags(node);
+            if (node.flags & NodeFlags.Abstract) {

```

Comment:
I hadn't thought of this case and the more general behavior of exports? What would we like to support? Currently this is a parsing error.